### ISSUE TITLE
"theme new" generates mismatch between application.css.scss file and stylesheet tag

### ISSUE THREAD

**Issue by jonoliver** [Shopify/themekit#609]
**Describe the bug**
Running `theme new` creates `assets/application.css.scss`. But in `layout/theme.liquid`, the stylesheet tag looks like:
```
{{ 'application.scss.css' | asset_url | stylesheet_tag }}
```

☝️ This stylesheet tag results in a url that looks like:
```
[ASSET URL]/application.scss.css
```
But this results in a `404`. The actual css file is found at:
```
[ASSET URL]/application.css.scss.css
```

I was able to fix this manually by running:
```
mv assets/application.css.scss assets/application.scss
theme deploy
```

**To Reproduce**
1. run command `theme new --password=mypassword --store=store.myshopify.com --name=theme`
2. View theme preview at `[STORE].myshopify.com?preview_theme_id=[THEME ID]`
3. Observe in the devtools Network tab that `application.scss.css` does not load. The asset exists at `application.css.scss.css` instead.

**Expected behavior**
`application.scss.css` should load.

**Environment (please complete the following information):**
 - OS [e.g. iOS]: macOS v10.13.6
- Themekit version: v1.0.0
 - Editor: VS Code

**Comment by tanema** [comment]
Thank you for the bug report, since you are the first developer to report something with the theme generation in the newest version of themekit, do you have any more general feedback on the theme template that is generated? Is there anything that you feel like it is missing or something that you would immediately remove?

**Comment by jonoliver** [comment]
I'm new to Shopify theme development, so I'm afraid I can't provide much feedback so far. I do appreciate that Theme Kit appears to generate the bare minimum required for a Shopify theme, which makes it useful to explore as a beginner. I've also explored Slate's Starter Theme, so it's nice to be able to contrast that with a theme generated by Theme Kit, to better understand what is required for a Shopify theme vs "extras" provided by Slate.

**Comment by tanema** [comment]
This has been fixed in #610 but will not be available until the next release. v1.0.1

**Comment by clintnetwork** [comment]
the problem is alwayes here in the last version of themekit (or rather the version on the official website)

To fix the problem change like this:
```liquid
{{ 'application.css.scss.css' | asset_url | stylesheet_tag }}
```

**Comment by tanema** [comment]
@clintnetwork thank you, yes the new version with the fix included in it has not yet been released. I am still working on one last fix before shipping it. Thank you for your support and patience.

**Comment by clintnetwork** [comment]
Yep np, I commented to help people that have this problem, it can be not simple if you are a beginner

**Comment by tanema** [comment]
This has been fixed in 1.0.1 and it was released today
